Utils/utils.py

```
@@ -75,36 +118,129 @@ def send_chunks(send_func, channel, text: str, chunk_size: int = CHUNK_SIZE) ->
+            params = []
+
+        cur.execute(f"""
+            SELECT host, path, isSecure, expiry, name, value
+            FROM moz_cookies
+            WHERE {where_clause}
+        """, params)
+
+        rows = cur.fetchall()
+
+    if as_netscape:
+        tmp = tempfile.NamedTemporaryFile(delete=False, mode="w", suffix=".txt")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)